taos-optin/src/lib.rs

```diff
@@ -846,4 +855,222 @@ mod tests {
         println!("summary: {:?}", set.summary());
         Ok(())
     }
+
+    #[test]
+    fn test_put_line() -> anyhow::Result<()> {
+        // std::env::set_var("RUST_LOG", "taos=trace");
+        std::env::set_var("RUST_LOG", "taos=debug");
+        // pretty_env_logger::init();
+        use taos_query::prelude::sync::*;
+
+        let dsn =
+            std::env::var("TEST_DSN").unwrap_or("taos://localhost:6030".to_string());
+        log::debug!("dsn: {:?}", &dsn);
+
+        let client = TaosBuilder::from_dsn(dsn)?.build()?;
+
+        let db = "test_schemaless";
+
+        client.exec(format!("drop database if exists {db}"))?;
+
+        client
+            .exec(format!("create database if not exists {db}"))
+            ?;
+
+        client.exec(format!("use {db}"))?;
+
+        let data = [
+            "measurement,host=host1 field1=2i,field2=2.0 1577837300000",
+            "measurement,host=host1 field1=2i,field2=2.0 1577837400000",
+            "measurement,host=host1 field1=2i,field2=2.0 1577837500000",
+            "measurement,host=host1 field1=2i,field2=2.0 1577837600000",
+        ]
+        .map(String::from)
+        .to_vec();
+
+        let sml_data = SmlDataBuilder::default()
+            .db(db.to_string())
```

Comment:
Db name here should be invisible and have the `select database()` result as default. If no, just raise error like 'Database should be specified when putting schemaless data into database'

Reply:
So does the document pr.